scripts/e2e-mobile.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+export SAUCE_USERNAME=okta-qa
+export SAUCE_ACCESS_KEY="$(aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/saucelabs/saucelabs_access_key /dev/stdout)"
+export SAUCE_PLATFORM_NAME="iOS";
+aws s3 --quiet --region us-east-1 cp s3://ci-secret-stash/prod/signinwidget/test_credentials ./test_credentials.yaml
```

Comment:
yaml file with credentials needed to run e2e tests are stored in this aws bucket.

```
WIDGET_TEST_SERVER: redacted
WIDGET_BASIC_USER: redacted
WIDGET_BASIC_PASSWORD: redacted
...
```